File: apps/s-2-admin/src/api/team/Team.ts
import { EventType } from "../eventType/EventType";
import { Membership } from "../membership/Membership";

export type Team = {
  id: number;
  name: string | null;
  slug: string | null;
  logo: string | null;
  bio: string | null;
  hideBranding: boolean;
  eventTypes?: Array<EventType>;
  members?: Array<Membership>;
};

File: apps/cactus-demo/src/APIs/Team/Dtos/Team.cs
namespace CactusDemo.APIs.Dtos;

public class Team
{
    public int Id { get; set; }

    public string? Name { get; set; }

    public string? Slug { get; set; }

    public string? Logo { get; set; }

    public string? Bio { get; set; }

    public bool HideBranding { get; set; }

    public List<int>? EventType { get; set; }

    public List<int>? Membership { get; set; }
}
